### ISSUE TITLE
use iso_fortran_env gives a segfault in F_Front

### ISSUE THREAD

**Issue by clementval** [C2SM-RCM/claw-compiler#211]
```fortran 
program original_code
  use iso_fortran_env
end program original_code
```

```bash
clawfc --force -o transformed_code.f90 original_code.f90 
```

```bash
clawfc: line 334: 92700 Segmentation fault: 11  ${OMNI_F2X_CMD} "${include_opt[@]}" "${module_opt[@]}" "${frontend_add_opt[@]}" ${OMNI_F2X_OPT} "${file_pp}" -o "${file_in_x}"
```

Abort signal is throw at line 751 in F-output-xcodeml.c
Probably the module is not read correctly.

**Comment by clementval** [comment]
Fix with PR#182 on OMNI Compiler https://github.com/omni-compiler/omni-compiler/pull/182

**Comment by clementval** [comment]
PR has been merged and OMNI updated for CLAW
